src/App.tsx

```diff
@@ -3,16 +3,26 @@ import Home from "./pages/Home";
 import SignUp from "./pages/SignUp";
 import SignIn from "./pages/SignIn";
 import ForgotPassword from "./pages/ForgotPassword";
+import Theme from "./theme";
+import styled from "styled-components";
 
 function App() {
   // theme provider here
+  <Theme>
+    <div>
+      <h2>Test</h2>
+    </div>
+  </Theme>
```

Comment:
you dont need this anymore. It's probably giving you error cause this should be inside a return.

Reply:
Removed!